pkg/proto/datadog/languagedetection/api.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
Could you update the .github/CODEOWNER so your team owns this file (similar to https://github.com/DataDog/datadog-agent/blob/main/.github/CODEOWNERS#L330) ?

Reply:
Ah yes absolutely! Updated